gcloud-java-dns/src/test/java/com/google/gcloud/dns/OptionTest.java

```diff
@@ -54,11 +52,11 @@ public void testConstructor() {
     assertEquals(RPC_OPTION, OPTION.rpcOption());
     assertEquals(VALUE, OPTION.value());
     try {
-      new AbstractOption(null, VALUE) {};
+      new Option(null, VALUE) {};
       fail("Cannot build with empty option.");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [In JUnit4, use the @Test(expected) annotation to denote tests that should throw exceptions](https://www.codacy.com/app/mziccard/gcloud-java/file/2326222385/issues/source?bid=3123103&fileBranchId=3204660#l56)